packages/request-manager/src/utils.ts

```diff
@@ -0,0 +1,39 @@
+import { createTimeoutPromise } from '@trezor/utils';
+
+export const waitUntil = (
```

Comment:
This is in my opinion almost the same thing as `scheduleAction` util with `attempts` and `gap` params, with exception of error concatenation - `scheduleAction` throws only the last one. Which may be important, dunno.

Reply:
It definitely makes sense to use an util that we use more broadly and it is tested. What about https://github.com/trezor/trezor-suite/pull/15642/commits/b60c7e9a45c6c17c7f17e6b506c6aef493c6c0fd  ? 
Let me know if there is a better way to use `scheduleAction`.